ui/components/src/EChart/EChart.tsx

```diff
@@ -15,7 +15,7 @@ import React, { useEffect, useLayoutEffect, useRef } from 'react';
 import { ECharts, EChartsCoreOption, init } from 'echarts/core';
 import { Box, SxProps, Theme } from '@mui/material';
 import { isEqual, debounce } from 'lodash-es';
-import { EChartsTheme } from './model';
+import { EChartsTheme } from '../model';
```

Comment:
Changed because I moved `ECharts` into a directory for consistency with the rest of our components.